app/assets/stylesheets/header.scss

```diff
@@ -13,35 +21,36 @@
   margin-right: 50px;
 }
 .top-nav ul{
-  padding:0;
-  margin:0;
+  padding: 0;
+  margin: 0;
 }
 .top-nav ul li{
-  display:inline-block;
+  display: inline-block;
   margin: .5em 2em 0;
 }
 .top-nav ul li a{
-  color:#97999B;
-  font-size:17px;
-  transition:.5s all;
-  -webkit-transition:.5s all;
-  -moz-transition:.5s all;
-  -o-transition:.5s all;
-  -ms-transition:.5s all;
+  color: #686c6d;
```

Comment:
Color literals like `#686c6d` should only be used in variable declarations; they should be referred to via variable everywhere else.